Makefile

```
@@ -1,4 +1,5 @@
-roms := pokecrystal.gbc pokecrystal11.gbc pokecrystal_au.gbc pokecrystal_debug.gbc pokecrystal11_debug.gbc
+patches := pokecrystalvc.patch
+roms    := pokecrystal.gbc pokecrystal11.gbc pokecrystal_au.gbc pokecrystal_debug.gbc pokecrystal11_debug.gbc
```

Just putting the ROMs first since they're our primary "product".
```suggestion
roms    := pokecrystal.gbc pokecrystal11.gbc pokecrystal_au.gbc pokecrystal_debug.gbc pokecrystal11_debug.gbc
patches := pokecrystalvc.patch
```